Documentation/quickstart/debugging.rst

```diff
@@ -49,30 +54,30 @@ Changing Debug Settings Quickly
 You can use the ``kconfig-tweak`` script that comes with the ``kconfig-frontends`` tools to quickly change debug settings,
 for instance turning them on or off before doing a build:
 
-.. code-block:: bash
+.. code-block:: console
 
    $ kconfig-tweak --disable CONFIG_DEBUG_NET
    $ kconfig-tweak --enable CONFIG_DEBUG_NET
 
 You can put a bunch of these into a simple script to configure the logging the way you want:
 
-.. code-block:: bash
+.. code-block:: console
 
    #!/bin/bash
 
-   $ kconfig-tweak --disable CONFIG_DEBUG_ALERT
-   $ kconfig-tweak --disable CONFIG_DEBUG_FEATURES
-   $ kconfig-tweak --disable CONFIG_DEBUG_ERROR
-   $ kconfig-tweak --disable CONFIG_DEBUG_WARN
-   $ kconfig-tweak --disable CONFIG_DEBUG_INFO
-   $ kconfig-tweak --disable CONFIG_DEBUG_ASSERTIONS
-   $ kconfig-tweak --disable CONFIG_DEBUG_NET
-   $ kconfig-tweak --disable CONFIG_DEBUG_NET_ERROR
-   $ kconfig-tweak --disable CONFIG_DEBUG_NET_WARN
-   $ kconfig-tweak --disable CONFIG_DEBUG_NET_INFO
-   $ kconfig-tweak --disable CONFIG_DEBUG_SYMBOLS
-   $ kconfig-tweak --disable CONFIG_DEBUG_NOOPT
-   $ kconfig-tweak --disable CONFIG_SYSLOG_TIMESTAMP
+   kconfig-tweak --disable CONFIG_DEBUG_ALERT
```

Comment:
I think for `console` you need the prompt. Otherwise it is assumed to be output of a command and it appears dimmer.

Reply:
@v01d Ok, thanks for catching. I accepted your changes.